src/woo_publications/metadata/tests/test_informatie_categorie_admin.py

```diff
@@ -0,0 +1,212 @@
+        url = reverse("admin:metadata_informatiecategorie_changelist")
+
+        with self.subTest("filter_on_naam"):
+            response = self.app.get(url, {"q": "first item"}, user=self.user)
+
+            self.assertEqual(response.status_int, 200)
+            # first item in list
+            self.assertInHTML(
+                '<td class="field-naam">first item</td>',
+                str(response.content),
+            )
+            # second item not in list
+            with self.assertRaises(AssertionError):
+                self.assertInHTML(
+                    '<td class="field-naam">second item</td>',
+                    str(response.content),
+                )
+
+        with self.subTest("filter_on_identifier"):
+            response = self.app.get(
+                url, {"q": "https://www.example.com/waardenlijsten/2"}, user=self.user
+            )
+
+            self.assertEqual(response.status_int, 200)
+            # first item in list
+            self.assertInHTML(
+                "https://www.example.com/waardenlijsten/2",
+                str(response.content),
+            )
+            # second item not in list
+            with self.assertRaises(AssertionError):
+                self.assertInHTML(
+                    "https://www.example.com/waardenlijsten/1",
+                    str(response.content),
+                )
+
+        with self.subTest("filter_on_oorsprong"):
+            response = self.app.get(
+                url,
+                {"oorsprong__exact": InformatieCategorieOrigins.value_list},
+                user=self.user,
+            )
+
+            self.assertEqual(response.status_int, 200)
+            # first item in list
+            self.assertInHTML(
+                "https://www.example.com/waardenlijsten/1",
+                str(response.content),
+            )
+            # second item not in list
+            with self.assertRaises(AssertionError):
+                self.assertInHTML(
+                    "https://www.example.com/waardenlijsten/2",
+                    str(response.content),
+                )
+
+    def test_informatie_category_admin_can_not_update_item_with_oorsprong_waardenlijst(
+        self,
+    ):
+        url = reverse(
+            "admin:metadata_informatiecategorie_change",
+            kwargs={"object_id": self.information_category_value_list.id},
+        )
+
+        response = self.app.get(url, user=self.user)
+        form = response.forms["informatiecategorie_form"]
+
+        self.assertEqual(response.status_code, 200)
+        # page doesn't have save button.
+        with self.assertRaisesMessage(
+            AssertionError,
+            'Couldn\'t find \'<input class="default" name="_save" type="submit" value="Opslaan">\' in response',
+        ):
+            self.assertInHTML(
+                '<input type="submit" value="Opslaan" class="default" name="_save">',
+                str(response.content),
+            )
```

Comment:
this is a very fragile way of asserting the behaviour, and moreover, it doesn't actually validate that the backend blocks the save, it just checks that a button is missing.

Better is to `form.submit(name="_save")` and assert that you get some kind of error response (I expect an HTTP 403, but it might be something else) *and* assert that the model instance was not updated.